aten/src/ATen/Context.h

```diff
@@ -126,6 +126,19 @@ class TORCH_API Context {
   bool deterministicCuDNN() const;
   void setDeterministicCuDNN(bool);
 
+  // Note [Disabling Fused SDP Kernels]
+  // ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+  // Fused SDP kernels are enabled by default. However, they can be disabled
+  // by setting the at::globalContext().setDisableFusedSDP(true) flag.
+  // This is useful for debugging purposes. For example, if you want to
+  // compare the performance of the fused SDP kernels with the unfused
+  // kernels, you can disable the fused SDP kernels.
```

Comment:
You can also force your code to use fused  kernels only if you disable the math kernels.

Reply:
will add